app/uk/gov/hmrc/minorentityidentificationfrontend/connectors/RegistrationConnector.scala

```diff
@@ -31,14 +31,29 @@ class RegistrationConnector @Inject()(httpClient: HttpClient,
                                       appConfig: AppConfig
                                      )(implicit ec: ExecutionContext) {
 
-  def register(sautr: String, regime: String)(implicit hc: HeaderCarrier): Future[RegistrationStatus] = {
+  def registerTrust(sautr: String, regime: String)(implicit hc: HeaderCarrier): Future[RegistrationStatus] = {
 
     val jsonBody = Json.obj(
       "sautr" -> sautr.toUpperCase,
       "regime" -> regime
     )
 
-    httpClient.POST[JsObject, RegistrationStatus](appConfig.registerUrl, jsonBody)(
+    httpClient.POST[JsObject, RegistrationStatus](appConfig.registerTrustUrl, jsonBody)(
```

Comment:
Hey Refaque can we move the POST method in a private method the only thing that it will take is a url and a jsonBody? What do you think?